libs/ui-toolkit/src/components/dropdown-menu/dropdown-menu.tsx

```diff
@@ -73,7 +73,7 @@ export const DropdownMenuContent = forwardRef<
   <DropdownMenuPrimitive.Content
     {...contentProps}
     ref={forwardedRef}
-    className="min-w-[290px] bg-neutral-200 dark:bg-white p-2 rounded z-20 dark:text-black"
+    className="min-w-[290px] bg-neutral-100 dark:bg-neutral-900 p-2 rounded z-20 text-black dark:text-white dark:border-neutral-700 border border-neutral-300"
```

Comment:
chore (blocking): We should be using the vega gray scale here. The border color should be `border-vega-light-200 dark:border-vega-dark-200`

Reply:
updated to use vega-light and vega-dark scale